test/e2e/checkpoint_test.go

```diff
@@ -1339,4 +1339,40 @@ var _ = Describe("Podman checkpoint", func() {
 		Expect(podmanTest.NumberOfContainersRunning()).To(Equal(0))
 	})
 
+	It("podman checkpoint and restore container with --file-locks", func() {
+		if !strings.Contains(podmanTest.OCIRuntime, "runc") {
+			// FIXME: Enable crun test when this feature has been released
+			// https://github.com/containers/crun/pull/783
+			Skip("Test only works with runc.")
```

Comment:
Could you change this to `Skip("FIXME: requires crun >= 1.4")`? I have a `find-obsolete-skips` script that I run from time to time, and a clear FIXME would make it easy for me to remove this in the next few months.

Reply:
Thanks, I've updated the skip message.